docs/benchmarks/graph-analytics.md

```diff
@@ -29,8 +29,9 @@ The benchmark runs the PageRank algorithm on GraphX through the spark-submit scr
 
 For example, to ensure that Spark has enough memory allocated to be able to execute the benchmark in-memory, supply it with --driver-memory and  --executor-memory arguments:
 
-    $ docker run --rm --volumes-from twitter-data cloudsuite/graph-analytics:4.0 \
+    $ docker run --rm --volumes-from twitter-data -e WORKLOAD_NAME=pagerank cloudsuite/graph-analytics:4.0 \
                  --driver-memory 4g --executor-memory 4g
+The container can also run `connected components` and `triangle count` algorithms on the given dataset by setting the environment variable `WORKLOAD_NAME` to `cc` and `tc`, respectively. Allocate over 40g for the `driver-memory` if you run the triangle count algorithm on the twitter dataset.
```

Comment:
I think it would be better if we keep the name convention consistent. If you use short name for connected components and the triangle count, you should also use the short name for the page rank like `pg`.

Reply:
Also, I don't think 40GB is required if I restrict the application running on 1 core by setting cpusets. We should mention that the memory size is equivalent to the number of cores involved.

Reply:
Did you check how much memory you required to run the tc benchmark with a single core?